packages/cli/internal/pkg/cli/context_deploy.go

```diff
@@ -81,7 +86,12 @@ func (o *deployContextOpts) Execute() ([]context.Detail, error) {
 		contextDetails[i] = result.info
 	}
 	if hasErrors {
-		return nil, fmt.Errorf("one or more contexts failed to deploy")
+		aggregateSuggestions = slices.DeDuplicateStrings(aggregateSuggestions)
+		return nil, clierror.New(
+			"agc context deploy",
+			o.deployContextVars,
+			actionableerror.New(fmt.Errorf("one or more contexts failed to deploy"), strings.Join(aggregateSuggestions, ", ")),
```

Comment:
I like the general approach here but it might make sense to simplify the error a little bit as the following is a little confusing
```
Error: an error occurred invoking 'context deploy'
with variables: {contexts:[] deployAll:true}
caused by: an error occurred invoking 'agc context deploy'
with variables: {contexts:[spotCtx onDemandCtx] deployAll:true}
caused by: one or more contexts failed to deploy
suggestion: Please ensure your account is activated by running 'agc account activate'
```

Could we maybe pass back this line and then print to the console on line 145?